acceptance/lib/helper.rb

```diff
@@ -23,18 +23,22 @@ def self.initialize_config(options)
                          nil, "Puppet Server Version",
                          "PUPPETSERVER_VERSION", nil, :string)
 
+    # This value now come from the beaker options file
+    #   the beaker options file can have this value updated via the rake task
     puppet_version = get_option_value(options[:puppet_version],
                          nil, "Puppet Version", "PUPPET_VERSION",
-                         "4.99.0.403.g33cc892",
+                         nil,
                          :string) ||
                          get_puppet_version
 
     # puppet-agent version corresponds to packaged development version located at:
     # http://builds.delivery.puppetlabs.net/puppet-agent/
+    # This value now come from the beaker options file
+    #   the beaker options file can have this value updated via the rake task
```

Comment:
This is something that might be out of the scope of this PR (ie you don't have to clean up every little thing in our acceptance helpers that you could after this change), but do we need these any more? Should we simply remove them from here or does that break something? Do you think this is valuable as documentation?